includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Advanced Settings Fields file.
+ *
+ * @package ActivityPub
+ */
+
+namespace ActivityPub\WP_Admin;
+
+/**
+ * Advanced Settings Fields class.
+ */
+class Advanced_Settings_Fields {
+
+	/**
+	 * Initialize.
+	 */
+	public static function init() {
+		add_action( 'load-settings_page_activitypub', array( self::class, 'register_advanced_fields' ) );
```

Comment:
There's an opportunity to prefix function calls with backslashes here and in later function calls.